packages/component-library/stories/README.md

```diff
@@ -0,0 +1,85 @@
+# CIVIC Design System Story Pattern
+
+Storybook is used to document [CIVIC's Design System](https://hackoregon.github.io/civic/). Each component in the library has a story.
+
+As you read through this document, refer to these stories to see the design pattern in use:
+
+- [Horizontal Bar Chart](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-horizontal-bar-chart--standard)
+- [Line Chart](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-line-chart--standard)
+- [Scatterplot](https://hackoregon.github.io/civic/?path=/story/component-lib-charts-scatterplot--standard)
+- [Scatterplot Map](https://hackoregon.github.io/civic/?path=/story/component-lib-maps-scatterplot-map--standard)
+
+Note: To bring up the Addons Panel, select it from the dropdown list in the upper left corner of the screen, next to the CIVIC logo.
+
+## Stories
+
+There are three types of stories:
+
+- Standard
+- Custom
+- Example(s)
+
+### Standard Story
+
+The Standard story is the default configuration of the component. For consistency, this is the configuration that is used unless it doesn't meet the needs of your project.
+
+### Custom Story
+
+The Custom story provides knobs for all the props that are available for this component. This gives you the ability to design any possible configuration of the component.
+
+### Example Story(s)
+
+If there is a configuration of the component that has been used in a project, and may be useful again, create an example story.
+
+## Knobs
```

Comment:
Is it worth mentioning storybook/addon-actions in addition to knobs? I'm trying to think which things might have saved me time, but again, I don't know how much the Dropdown component is an anomaly compared to others. 🤷‍♀

Reply:
I think mentioning actions would be helpful. I know that is something I had to look up when I first started working in Storybook.

Reply:
Thank you for the feedback!  I listed the types of knobs, currently.  The two references to Knobs have examples of their use... also the stories we've done that I've referenced have examples.  I think this will be enough... let me know if you disagree.

I added a section on the Actions addon... good suggestion.  Actions are useful for the input  components, which we are adding to Storybook.